GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GitReadAndGitLockTests.cs

```diff
@@ -63,7 +63,8 @@ public void GitAliasNamedAfterKnownCommandAcquiresLock()
             string alias = nameof(this.GitAliasNamedAfterKnownCommandAcquiresLock);
 
             int pid;
-            GitHelpers.AcquireGVFSLock(this.Enlistment, out pid, resetTimeout: 3000);
+            int tenSeconds = 10 * 1000;
+            GitHelpers.AcquireGVFSLock(this.Enlistment, out pid, resetTimeout: tenSeconds);

```

Comment:
Why did this need to be increased?

Reply:
The timeout of 3 secs occasionally passes before the `GitReadAndGitLockTests` could run Git commands against the locked repo. As a result the test commands run in the enlistment  root succeed and the expected command output `Waiting for 'GVFS.FunctionalTests.LockHolder` does not show in command output. This causes the tests to fail. 10 secs seem to be adequate.